src/main/scala/mesosphere/marathon/api/akkahttp/v2/PodsController.scala

```diff
@@ -85,7 +85,30 @@ class PodsController(
       }
     }
 
-  def update(podId: PathId): Route = ???
+  @SuppressWarnings(Array("all")) // async/await
+  def update(podId: PathId): Route = {
+    authenticated.apply { implicit identity =>
+      (entity(as[raml.Pod]) & forceParameter & extractClientIP & extractUri) {
+        case (ramlPod, force, host, uri) =>
+          normalized(ramlPod, podNormalizer) { normalizedPodDef =>
```

Comment:
I think you miss the normal validation before. Or is this done in `entity[as[raml.Pod]]`?

Reply:
Ok, apparently we used the implicit validation in `create` already. So this should be fine.